spec/factories/assignments.rb

```diff
@@ -22,6 +22,7 @@
     course_id 481
     article_id 124_884_99
     article_title 'Siderocalin'
+    wiki
```

Comment:
Weird indeed. Maybe factory_girl does something different if you don't include a value? Maybe we should set an id for the wiki fixture, and set wiki_id to that in this factory.